### DISCUSSION TITLE
Improvement suggestion: Thumbnail cell in browser should be square

### DISCUSSION THREAD

**Discussion by kenerling** [Beep6581/RawTherapee#6882] [Ideas]
Hello to all,

First post ever here, so I hope I'm in the right place.

I'd like to suggest that in RawTherapee's browser, the thumbnail cells should be square.

The cells are currently rectangular, which "favors" landscape-oriented images, that is, these latter appear bigger in the browser than do their portrait-oriented counterparts. Because the landscape-oriented images appear bigger, the eye is drawn to them more than it is the portrait-oriented ones.

Making the thumbnail cell square would equalize the size of landscape and portrait images, thus putting them on equal ground. The browser would thus function just like slides on a light-table.

Thanks for your consideration, and happy shooting to all.

**Comment by Beep6581** [comment]
Hey

There are already open feature requests with lengthy discussions on the matter, e.g. #1239
